simphony/io/tests/test_h5_lattice.py

```diff
@@ -73,9 +108,10 @@ def cleanup(self):
         shutil.rmtree(self.temp_dir)
 
     def container_factory(self, name, type_, base_vect, size, origin):
-        return H5Lattice.create_new(
-            self.handle.root, 'foo', type_, base_vect,
-            size, origin, record=CustomRecord)
+        self.group = self.handle.create_group(
```

Comment:
what about making the following changes to avoid `self.group`. 
one could use H5Cuds:
```self.handle = H5CUDS.open(self.filename)```
and then, change  `container_factor` to
```
    def container_factory(self, name, type_, base_vect, size, origin):
        self.name = name
        return self.handle.add_lattice(Lattice(name,
            type_, base_vect, size, origin))
```
`self.group` is removed and the spot where self.group was used could be updated to
```
    def test_initialization_from_existing_lattice_in_file(self):
        """ Checks that H5Lattice constructed from Lattice with a
        CustomRecord column description has correct attributes
        """
        lattice = self.handle.get_lattice(self.name)
        self.assertEqual(lattice.name, 'foo')
        ...
```